### DISCUSSION TITLE
Routing on Omaps for Mac

### DISCUSSION THREAD

**Discussion by pinacate** [organicmaps/organicmaps#5048] [Q&A] [answered]
I build Omaps for my Mac. The app runs and looks great on the Mac. However when I push the routing icon (blue arrow) on the action strip it doesn't enter routing mode as I'd expected. The other icons/features work fine. 

Does anyone know why this is?

Thanks for any input.

**Comment by hb0nd** [comment, marked as answer]
If you mean a live navigation view like on mobile, that isn't implemented in desktop yet afaik. 

To select the start and end points for a route, you need to select which point to place in the menu, then `hold shift` and press `left click` to add the point:

https://user-images.githubusercontent.com/26939824/235002745-b8d7b293-30aa-41fe-a747-8b8d9887fc44.mp4

(it's a bit cumbersome, would be good to rework it)
